statistika-statistics-and-economy-journal.csl

```diff
@@ -0,0 +1,247 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never" default-locale="en-US">
+  <info>
+    <title>Statistika: Statistics and Economy Journal</title>
+    <id>http://www.zotero.org/styles/statistika-statistics-and-economy-journal</id>
+    <link href="http://www.zotero.org/styles/statistika-statistics-and-economy-journal" rel="self"/>
+    <link href="http://www.zotero.org/styles/journal-of-urban-and-environmental-engineering" rel="template"/>
+    <link href="https://www.czso.cz/csu/czso/guide_for_authors" rel="documentation"/>
+    <author>
+      <name>Libor Ansorge</name>
+      <email>libor.ansorge@gmail.com</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="science"/>
+    <issn>0322-788x</issn>
+    <eissn>1804-8765</eissn>
+    <updated>2016-04-19T17:32:13+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+    <locale>
+    <terms>
+      <term name="cited" form="short">cit</term>
+    </terms>
+  </locale>
+  <!-- MACRO SECTION -->
+  
+  <macro name="author">
+    <names variable="author">
+      <name name-as-sort-order="all" sort-separator=", " initialize-with="." delimiter=", " delimiter-precedes-last="never">
+        <name-part name="family" text-case="uppercase"/>
+        <name-part name="given"/>
+      </name>
+      <label form="short"/>
+      <substitute>
+        <names variable="editor"/>
+        <names variable="translator"/>
+        <text macro="title"/>
+      </substitute>
+    </names>
+  </macro>
+  
+  <macro name="author-short">
+    <names variable="author">
+      <name form="short" delimiter=", " delimiter-precedes-last="never"/>
+      <substitute>
+        <names variable="editor"/>
+        <names variable="translator"/>
```

Comment:
I'd assume you want `<text macro="title"/>` here too? Otherwise author and author-short macro don't align (and you'd get just the year for anonymous citations).